test/IntegrationTests/AzureTests.cs

```diff
@@ -48,6 +46,10 @@ public void SubmitsTraces(string packageVersion)
         collector.Expect("System.Net.Http");
 #elif NET6_0_OR_GREATER
         collector.Expect("OpenTelemetry.Instrumentation.Http.HttpClient");
+#elif NETFRAMEWORK
+        // On .NET Framework the "OpenTelemetry.Instrumentation.Http.HttpWebRequest"
+        // ends up being suppressed by the addition of headers via the Azure instrumentation
+        // See https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs#L279-L284
```

Comment:
Link to main can be desynchronized. Please use commit or tag

```suggestion
        // See https://github.com/open-telemetry/opentelemetry-dotnet/blob/f6a1c04e8115a828dd33269f639daf2924796bae/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs#L279-L284
```

Reply:
913501fd894e6e02dfe64d9bc0732979c7dcc1ce